utils/coreutils/techutils.go

```diff
@@ -29,13 +29,30 @@ const (
 	Go     Technology = "go"
 	Pip    Technology = "pip"
 	Pipenv Technology = "pipenv"
+	Pypi   Technology = "pypi"
```

Comment:
Pypi is not a technology.
we have 3 tools for python: `Pip`, `Pipenv` and `Poetry`
```suggestion
	Pypi = "pypi"
```